webclient/components/1time-Mrow-mField/diagnosis/view-layer/grid-with-cards-ct.vue

```diff
@@ -45,48 +58,80 @@
 </template>
 
 <script>
+import moment from 'moment'
 import clientSideTblPatientDiagnosis from '../db/client-side/structure/patient-table-of-diagnosis'
+import clientSideTblMasterDiagnosis from '../db/client-side/structure/master-table-of-diagnosis'
 
 export default {
+  data() {
+    return {
+      defaultExpandAll: true,
+    }
+  },
   computed: {
     cfArOfDiagnosisForDisplay: function () {
       const getData = clientSideTblPatientDiagnosis
         .query()
         .with('linkWithMaster')
         .where('ROW_END', 2147483648000)
+        .where('masterDiagnosisId', (value) => value > 0)
         .get()
       return getData
     },
   },
   methods: {
+    expandRow(row, isExpanded) {
+      console.log('row', row);
+      return true;
+    },
+    fnFormatDateOfStartDate(time) {
+      return moment(time).format(
+        'MMM DD YYYY'
+      )
+    },
+
+    tableRowClassName({row, rowIndex}) {
+      if (row.vnRowStateInSession === 24) {
+        return 'warning-row';
+      } else if (row.vnRowStateInSession === 2456) {
+        return 'danger-row';
+      }
+      return '';
+    },
     mfIconDeleteClickedOnChildCard(id) {
       const arResultsFromOrm = clientSideTblPatientDiagnosis
         .query()
         .with('linkWithMaster')
         .where(id)
         .first()
 
-      this.$prompt(arResultsFromOrm.linkWithMaster.diagnosisName, 'Discontinue diagnosis', {
-        confirmButtonText: 'Discontinue',
+      this.$prompt(arResultsFromOrm.linkWithMaster.diagnosisName, 'Delete diagnosis', {
+        confirmButtonText: 'Delete',
         cancelButtonText: 'Cancel',
         inputPlaceholder: 'Enter reasen',
       })
         .then(async ({ value }) => {
           clientSideTblPatientDiagnosis.update({
             where: id,
             data: {
-              discontinueNote: value,
+              deleteNote: value,
               ROW_END: Math.floor(Date.now()),
             },
           })
+          await clientSideTblMasterDiagnosis.update({
+              where: (record) =>
+                record.masterDiagnosisId === arResultsFromOrm.masterDiagnosisId,
+              data: {
+                ROW_END: 2147483648000
+              },
+            })
           this.$message({
             type: 'success',
-            message: 'Diagnosis discontinue.',
+            message: 'Diagnosis deleted.',
           })
-          // console.log('delete status ======> ', value)
         })
         .catch(() => {
-          console.log('Discontinue cancelled')
+          console.log('Delete cancelled')
         })
     },
     fnOpenAddModule() {
```

Comment:
function name should be mfOpenAddCtInEditLayer

Reply:
Now I use the default add function for showContentInCardComponent